client/fingerprint_manager.go

```diff
@@ -22,8 +22,9 @@ type FingerprintManager struct {
 
 	// updateNode is a callback to the client to update the state of its
 	// associated node
-	updateNode func(*cstructs.FingerprintResponse) *structs.Node
-	logger     *log.Logger
+	updateNode        func(*cstructs.FingerprintResponse) *structs.Node
+	updateHealthCheck func(*cstructs.HealthCheckResponse) *structs.Node
```

Comment:
updateDrivers?

Reply:
I think the question is whether this should only be applied to drivers. For example, we might want to apply this to disk space. However, I wonder if something like disk space would only require the periodic fingerprinter.